### ISSUE TITLE
9.1 Metadata in forms

### ISSUE THREAD

**Issue by pbinkley** [ualbertalib/HydraNorth#30]
As a user, I want to be able to describe my deposited item according to ERA norms to enable discovery and reuse
	- start with one item type (image?)
	- Metadata team will provide specs on needed changes with examples to existing Sufia forms
Note: Full-featured image deposit will depend on #57
Note: Dependent on Data Dictionary #103
Done: 
- [x] have forms with all of the fields (as spec'ed by metadata team) needed to describe the items
- [x] stored properly in Fedora, indexed and searchable properly in Solr, 
- [x] stretch goal: integrate with external ontology e.g. geo names

**Comment by weiweishi** [comment]
I'm really interested in taking on this task. But it may require more than one pair of eyes from the dev team to get this working. Can we discuss the approach we'd like to take going forward on this issue?

**Comment by sfarnel** [comment]
maybe we can touch base at the Wed standup?

On Tue, Jan 20, 2015 at 12:33 AM, Weiwei Shi <notifications@github.com>
wrote:

> I'm really interested in taking on this task. But it may require more than
> one pair of eyes from the dev team to get this working. Can we discuss the
> approach we'd like to take going forward on this issue?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ualbertalib/HydraNorth/issues/30#issuecomment-70614723>
> .
>



-- 
Sharon Farnel
Metadata Coordinator
University of Alberta Libraries
sharon.farnel@ualberta.ca
780-492-3685

**Comment by weiweishi** [comment]
Some research on Authority Control in Sufia:
* questioning_authority: A gem used by the Hydra community to build local authority list, or a wrapper around existing authorities. Need to investigate more on how authority control information is handled in RDF and Fedora 4. 
* example to build drop down menu: https://github.com/curationexperts/mira/blob/5856e409d743fbc2e6fc026274972f5e10d4fb4e/app/helpers/contribute_helper.rb#L10
* example to get the long name (label) for the term (for example URI):
 https://github.com/curationexperts/mira/blob/276bdfe1ca3e2d465f6067a0dcc784bcac606efb/app/models/forms/capstone_project.rb#L18

**Comment by weiweishi** [comment]
@sfarnel Sufia uses Schema.org microdata to align with Blacklight which uses schema.org for bibliographic records. This includes the item types. Could you please review if my following map is correct:
    "Book" => "http://schema.org/Book",
    "Book Chapter" => "http://schema.org/Book",
    "Conference\/Workshop Poster" => "http://schema.org/CreativeWork",
    "Conference\/Workshop Presentation" => "http://schema.org/CreativeWork",
    "Image" => "http://schema.org/ImageObject",
    "Journal Article (Draft-Submitted)" => "http://schema.org/Article",
    "Journal Article (Published)" => "http://schema.org/Article",
    "Learning Object" => "http://schema.org/CreativeWork",
    "Map or Cartographic Material" => "http://schema.org/Map",
    "Report" => "http://schema.org/CreativeWork",
    "Research Material" => "http://schema.org/CreativeWork",
    "Review" => "http://schema.org/CreativeWork",
    "Thesis" => "http://schema.org/ScholarlyArticle",
    "Video" => "http://schema.org/VideoObject",
    "Other" => "http://schema.org/CreativeWork",

**Comment by sfarnel** [comment]
Looks good for the most part, though would suggest the following changes:

"Computing science technical report" => "http://schema.org/CreativeWork"
"Structural engineering report" => "http://schema.org/CreativeWork"
"Review" => "http://schema.org/Review"
"Thesis" => "http://schema.org/CreativeWork"

We don't have video in out item types yet, and research material is
essentially our 'other' category so we don't need that either.

On Tue, Jan 27, 2015 at 11:24 AM, Weiwei Shi <notifications@github.com>
wrote:

> @sfarnel <https://github.com/sfarnel> Sufia uses Schema.org microdata to
> align with Blacklight which uses schema.org for bibliographic records.
> This includes the item types. Could you please review if my following map
> is correct:
> "Book" => "http://schema.org/Book",
> "Book Chapter" => "http://schema.org/Book",
> "Conference\/Workshop Poster" => "http://schema.org/CreativeWork",
> "Conference\/Workshop Presentation" => "http://schema.org/CreativeWork",
> "Image" => "http://schema.org/ImageObject",
> "Journal Article (Draft-Submitted)" => "http://schema.org/Article",
> "Journal Article (Published)" => "http://schema.org/Article",
> "Learning Object" => "http://schema.org/CreativeWork",
> "Map or Cartographic Material" => "http://schema.org/Map",
> "Report" => "http://schema.org/CreativeWork",
> "Research Material" => "http://schema.org/CreativeWork",
> "Review" => "http://schema.org/CreativeWork",
> "Thesis" => "http://schema.org/ScholarlyArticle",
> "Video" => "http://schema.org/VideoObject",
> "Other" => "http://schema.org/CreativeWork",
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ualbertalib/HydraNorth/issues/30#issuecomment-71699653>
> .
>



-- 
Sharon Farnel
Metadata Coordinator
University of Alberta Libraries
sharon.farnel@ualberta.ca
780-492-3685

**Comment by pbinkley** [comment]
Note: we want to prevent users from applying multiple licenses.

**Comment by weiweishi** [comment]
sufia6.0.0.rc4 breaks the local batch update job overriding, which breaks how local metadata can be saved.  - currently job overriding works with sufia6.0.0.rc3 --https://github.com/projecthydra/sufia/commit/cbd0320ac8e8f480eb663de30e19e5b570282873

**Comment by pgwillia** [comment]
Can we close this?
